src/inference/include/ie/gpu/gpu_config.hpp

```diff
@@ -130,6 +130,13 @@ INFERENCE_ENGINE_1_0_DEPRECATED DECLARE_GPU_CONFIG_KEY(MAX_NUM_THREADS);
  * Thus, this key should be turned off if graph loading time is considered to be most important target to optimize.*/
 INFERENCE_ENGINE_1_0_DEPRECATED DECLARE_GPU_CONFIG_KEY(ENABLE_LOOP_UNROLLING);
 
+/**
+ * @brief Turning on this key disables winograd convolution in iGPU.
+ * Winograd convolution has different characteristics for accuracy and performance compared to other convolution
+ * implementations. Default value is false.
+ */
+INFERENCE_ENGINE_1_0_DEPRECATED DECLARE_GPU_CONFIG_KEY(DISABLE_WINOGRAD_CONVOLUTION);
```

Comment:
Don't add anything to the old api please

Reply:
removed it